spec/controllers/chargeback_controller_spec.rb

```diff
@@ -110,4 +111,25 @@
       expect(flash_array.first[:message]).to include("rate is assigned and cannot be deleted")
     end
   end
+  context "#get_cis_all" do
+    elements_miq = %w(enterprise storage ext_management_system ems_cluster tenants)
+    elements_miq.each do |element|
+      it "returns names of instances of " + element do
+        names = {}
+        classtype =
+          if element == "enterprise"
+            MiqEnterprise
+          else
+            element.classify.constantize
+          end
+
+        classtype.all.each do |instance|
+          names[instance.id] = instance.name
+        end
+        controller.instance_variable_set(:@edit, :new => {:cbshow_typ => element}, :cb_assign => {})
```

Comment:
@aljesusg 
there is problem that all models (expect to tenants) from elements_miqs don`t have created records in test db. so we are testing here mostly that {} is equal to {}.

To create these records you can use  
let!{ FactoryGirl.create(:miq_enterprise) }
let!{ FactoryGirl.create(:storage) } (`!` means evaluate it immediately)
etc. (for all these models from elements_miq)
and put it somewhere under `context`

thanks!